src/agent.rs

```diff
@@ -63,6 +63,7 @@ Note that there is an Oracle and Exporter for each network, but only one Local S
 ################################################################################################################################## */
 
 pub mod dashboard;
+pub mod holiday_hours;
```

Comment:
I think we can group holidays and market hours to each other. market hours seems like a good name, one will be weekly schedule, and the other will be something like holiday overwrites

Reply:
Done